docs/pipelines/multisample/shiva.md

```diff
@@ -82,6 +82,7 @@ unifiedgenotyper:
 ## Supported variant callers
 At this moment the following variant callers can be used
 
+##### Germline
```

Comment:
We may want to include a small section that this is "default/generic/regular" workflow.